app/services/ember-cli.js

```diff
@@ -80,7 +80,8 @@ const availableBlueprints = {
 const requiredDependencies = [
   'jquery',
   'ember',
-  'ember-template-compiler'
+  'ember-template-compiler',
+  'ember-data'
```

Comment:
Also, if a user left off ember-data, a gist is currently not loading. This is a bug.